cli/commands/config.go

```diff
@@ -2,6 +2,7 @@ package commands
 
 import (
 	"fmt"
+
```

Comment:
No, that's fine. 

Just wanted to make sure it was some automated tool doing it.